remote-api/src/main/java/org/alfresco/rest/api/impl/CategoriesImpl.java

```diff
@@ -187,6 +187,36 @@ public List<Category> linkNodeToCategories(final String nodeId, final List<Categ
         return categoryNodeRefs.stream().map(this::mapToCategory).collect(Collectors.toList());
     }
 
+    @Override
+    public void unlinkNodeFromCategory(final String nodeId, final String categoryId, Parameters parameters)
+    {
+        final NodeRef categoryNodeRef = getCategoryNodeRef(categoryId);
+        final NodeRef contentNodeRef = nodes.validateNode(nodeId);
+        verifyChangePermission(contentNodeRef);
+        verifyNodeType(contentNodeRef);
+
+        if (isCategoryAspectMissing(contentNodeRef))
+        {
+            throw new InvalidArgumentException("Node with id: " + nodeId + "does not belong to a category");
```

Comment:
Minor nit pick - missing space:
```suggestion
            throw new InvalidArgumentException("Node with id: " + nodeId + " does not belong to a category");
```